### DISCUSSION TITLE
OTP/TOTP Question

### DISCUSSION THREAD

**Discussion by jflowers1974** [joeycastillo/Sensor-Watch#98] [General]
What's the workflow / process to get this functionality working? I see some folks talking about NFC to get the necessary info shared in the first place.  However, that route seems to require additional hardware.  Is OTP supported 'out of the box' or does it require additional hacking?

**Comment by wryun** [comment]
https://blog.singleton.io/posts/2022-10-17-otp-on-wrist/ (not me, @dps)

i.e. basically build a new firmware with your keys baked in.

If you want something that's a bit less manual, I've raised https://github.com/joeycastillo/Sensor-Watch/pull/95 (untested as yet on real hardware), but this will require a serial connection to the watch to copy the TOTP URIs across. If you use Aegis, they're easy to get/extract.
